src/script/interpreter.cpp

```diff
@@ -14,6 +15,9 @@
 
 typedef std::vector<unsigned char> valtype;
 
+static_assert(static_cast<uint64_t>(std::numeric_limits<int64_t>::max()) == UINT64_C(0x7FFFFFFFFFFFFFFF));
```

Comment:
Need a comment as to why we are making these asserts.  Something about that these values are consensus critical and used in the Elements 64-bit arithmetic opcodes.